doc/releases/changelog-dev.md

```diff
@@ -212,6 +211,9 @@
 * `qml.transforms.hamiltonian_expand` can now handle multi-term observables with a constant offset.
   [(#5414)](https://github.com/PennyLaneAI/pennylane/pull/5414)
 
+* The `qml.qchem.hf_state` function is upgraded to be compatible with parity and Bravyi-Kitaev basis.
```

Comment:
```suggestion
* The `qml.qchem.hf_state` function is upgraded to be compatible with the parity and Bravyi-Kitaev bases.
```

Reply:
I think it is supposed to be basis not bases.

Reply:
:thinking: Not a big deal, but I'd be fine with:
- upgraded to be compatible with the parity basis and Bravyi-Kitaev basis
- upgraded to be compatible with the parity and Bravyi-Kitaev bases

In the latter, the `bases` refers to both `parity` and `Bravyi-Kitaev` so hence I think it should be plural.

Or are `parity` and `Bravyi-Kitaev` _the same basis_? In which case I agree with you.

Reply:
@ddhawan11 Please use _bases_ as Tom originally suggested.

Reply:
@trbromley It is changed to _bases_ as you suggested.